tests/units/classes/mock/streams/fs/file.php

```diff
@@ -412,7 +412,7 @@ public function testFseek()
 			->then
 				->boolean($fileObject->eof())->isTrue()
 				->boolean($fileObject->valid())->isFalse()
-				->string($fileObject->current())->isEmpty()
+				->variable($fileObject->current())->isEqualTo('')
```

Comment:
Because PHP are not consistent here : PHP returns an empty string while HHVM returns false.